packages/dashboard-frontend/src/Layout/Header/Tools/index.tsx

```diff
@@ -204,7 +204,7 @@ export class HeaderTools extends React.PureComponent<Props, State> {
                 Click here
               </Button>
               <span> to see the login command and copy it manually.</span>
-              <pre className="refresh-token-area">{loginCommand}</pre>
+              <pre className='refresh-token-area'>{loginCommand}</pre>
```

Comment:
As for me, I used " instead of ' in HTML. 
@akurinnoy WDYT?

Reply:
yes, we use double quotes in HTML and single quotes in JS/TS.